Plugins/CoreAPI/server/graphql/schema/scalars.graphql

```diff
@@ -0,0 +1 @@
+scalar Date
```

Comment:
You don't need to add this since it's already in the [Server Schema](https://github.com/PLEXSolutions/PenPal/blob/master/Server/src/server/graphql/schema/schema.graphql#L6https://github.com/PLEXSolutions/PenPal/blob/master/Server/src/server/graphql/schema/schema.graphql#L6)

Reply:
rgr